openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -77,10 +77,12 @@ def _do_third_party_auth(request):
             provider_name=requested_provider.name,
         )
         message += "<br/><br/>"
-        message += _(
+        message += Text(_(
             "If you don't have an {platform_name} account yet, "
-            "click <strong>Register</strong> at the top of the page."
-        ).format(
+            "click {strong_start}Register{strong_end} at the top of the page."
+        )).format(
+            strong_start=HTML("<strong>"),
+            strong_end=HTML("</strong>"),
```

Comment:
@nasthagiri As I understand it, this is new code that you added two months ago.  The xssconfiglinter has forced me to address lint errors in this code, and I find that the above changes pass the linter.   However, I am not familiar with front end code, and I am concerned that wrapping this in Text() and HTML() routines should be done at a higher level.  I'm curious if you have any sense of how this HTML should be wrapped.